scheduler.py

```diff
@@ -121,4 +150,81 @@ def get_worker(self) -> Union[WorkerInfo, WorkerInfoWrapper]:
             else:
                 logging.info(f"No worker found.")
                 raise NoWorkerAvailableError(self.schedulerMode)
-        
\ No newline at end of file
+        elif self.schedulerMode == constants.SCHEDULINGMODE_LOADAWARE:
+            logging.info(f"Getting worker for {self.schedulerMode}")
+            wrappedWorkerInfo = self.get_worker_info(self.WorkersWithoutGPU, self.WorkersWithoutGPULock)
```

Comment:
why do we check WorkersWithoutGPU before WorkersWithGPU? This should depend on the type of the task?

Reply:
yeah this depends on type of task - given that isn't available right now, we assume all tasks do not need GPU